server/test/helpers/repositories/list.go

```diff
@@ -0,0 +1,97 @@
+package repositories
+
+import (
+	"database/sql"
+	"errors"
+)
+
+// ListOption defines dummy Option repository.
+type ListOption struct {
+	ID     int64
+	Action string
+}
+
+// DummyList defines dummy List repository.
+type DummyList struct {
+	ProjectID int64
+	UserID    int64
+	Option    *ListOption
+}
+
+// Find returns a list.
+func (d *DummyList) Find(projectID, id int64) (int64, int64, int64, sql.NullString, sql.NullString, sql.NullInt64, bool, error) {
+	return id, projectID, d.UserID, sql.NullString{String: "title", Valid: true}, sql.NullString{String: "#ff0000", Valid: true}, sql.NullInt64{Int64: d.Option.ID, Valid: true}, false, nil
+}
+
+// Find returns a list.
```

Comment:
__[Golint]__ comment on exported method DummyList.FindByTaskID should be of the form "FindByTaskID ..." ([link](https://sider.review/gh/repos/50192141/pulls/454#issue-3822500))

<sub>posted by Sider</sub>